invenio_rdm_records/services/customizations.py

```diff
@@ -41,57 +41,133 @@ def customize(cls, opts):
         return _make_cls(cls, attrs) if attrs else cls
 
 
-class FileConfigMixin:
-    """Shared customization for file service configs."""
+class FromConfig:
```

Comment:
``FromConfig*``s  ought to live in Invenio-Records-Resources so that it's reusable.